hudi-flink/src/main/java/org/apache/hudi/sink/partitioner/BucketAssignFunction.java

```diff
@@ -112,7 +113,7 @@
   /**
    * State to book-keep which partition is loaded into the index state {@code indexState}.
    */
```

Comment:
please update the doc as well

Reply:
partitionLoadState remaned as stateLoadFlag